modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -287,12 +313,29 @@ public void testGetObjectEntries() throws Exception {
 
 		Date date = RandomTestUtil.nextDate();
 
-		ObjectEntry objectEntry1 = _addObjectEntry("queued", date, title1);
+		DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(
+			"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+
+		LocalDateTime localDateTime1 = LocalDateTime.now();
+
+		LocalDateTime localDateTime2 = localDateTime1.plusHours(1);
+
+		String dateTimeString1 = dateTimeFormatter.format(
+			localDateTime1.withNano(0));
+
+		String dateTimeString2 = dateTimeFormatter.format(
+			localDateTime2.withNano(0));
+
+		ObjectEntry objectEntry1 = _addObjectEntry(
+			"queued", date, null, title1);
 		ObjectEntry objectEntry2 = _addObjectEntry(
-			"started", new Date(date.getTime() - Time.DAY), title2);
+			"started", new Date(date.getTime() - Time.DAY), localDateTime1,
+			title2);
 		ObjectEntry objectEntry3 = _addObjectEntry(
-			"completed", new Date(date.getTime() + Time.DAY), title3);
-		ObjectEntry objectEntry4 = _addObjectEntry("queued", date, title4);
+			"completed", new Date(date.getTime() + Time.DAY), localDateTime2,
+			title3);
+		ObjectEntry objectEntry4 = _addObjectEntry(
+			"queued", date, null, title4);
```

Comment:
Hey @jairmedeiros, could you please declare all variables as close to their usage as possible, for example:

````java
ObjectEntry objectEntry1 = _addObjectEntry(
	"queued", date, null, title1);

LocalDateTime localDateTime1 = LocalDateTime.now();

ObjectEntry objectEntry2 = _addObjectEntry(
	"started", new Date(date.getTime() - Time.DAY), localDateTime1,
	title2);

LocalDateTime localDateTime2 = localDateTime1.plusHours(1);

ObjectEntry objectEntry3 = _addObjectEntry(
	"completed", new Date(date.getTime() + Time.DAY), localDateTime2,
	title3);
ObjectEntry objectEntry4 = _addObjectEntry(
	"queued", date, null, title4);
````

Note that I did not include these things below because to follow the pattern it should be define right before their usages at [here](https://github.com/liferay-objects/liferay-portal/pull/3176/files#diff-82f88640eff05dedc724f92e76be2cf21cd6e4ffae22f9cba25e62e361ced5d0R435) for example

````java
DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(
			"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");

String dateTimeString1 = dateTimeFormatter.format(localDateTime1.withNano(0));

String dateTimeString2 = dateTimeFormatter.format(localDateTime2.withNano(0));
````

Thank you!